example.beancount

```diff
@@ -10,6 +10,10 @@ option "operating_currency" "USD"
 2010-01-01 custom "fava-extension" "fava_envelope" "{}"
 2010-01-01 custom "envelope" "start date" "2015-01"
 2010-01-01 custom "envelope" "budget account" "Assets:US:BofA:Checking"
+2015-01-01 custom "envelope" "repeating" "Expenses:Financial:Fees" 4
```

Comment:
doesn't appear to do anything with the current code, so it shouldn't be in the example.beancount

Reply:
agree